tasks/kibana.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Check for existing dashboards
+  uri:
+    url: "{{beats_client_kibana_url}}/_search?q=title:{{beats_client_kibana_dash_search[item]}}"
+    return_content: yes
+    body_format: json
+    body: {}
+  register: dashboard_check_result
+  changed_when: false
+  ignore_errors: yes
+  with_items: "{{ beats_client_beats_packages }}"
+
+- name: Inject dashboards into kibana index
+  command: /usr/share/{{item}}/scripts/import_dashboards -i {{beats_client_kibana_indices[item]}} {{beats_client_kibana_export_parameters}}
```

Comment:
Much more readable as: 

```yaml
- name: Inject dashboards into kibana index
  command: >-
    /usr/share/{{ item }}/scripts/import_dashboards
    -i {{ beats_client_kibana_indices[item] }}
    {{beats_client_kibana_export_parameters}}
```

Might submit a PR to ansible-lint for a) whitespace within handlebars interpolations and b) super-long-lines in commands.